PGM/src/main/java/tc/oc/pgm/menu/gui/SettingsTypeInterface.java

```diff
@@ -47,7 +28,7 @@ public void setButtons() {
         List<Button> buttons = new ArrayList<>();
         buttons.add(new Button(
                 new ItemCreator(Material.BOOK_AND_QUILL)
-                        .setName(Constants.PREFIX + new Component(new TranslatableComponent("setting.types.chat")).getText())
+                        .setName(Constants.PREFIX + PGMTranslations.get().t("setting.types.chat", player))
```

Comment:
should probs try to use translatable components for all the buttons, and names, and pretty much everything. but it's fine